tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -213,14 +213,31 @@ drop_chunks_add_policy(PG_FUNCTION_ARGS)
 Datum
 drop_chunks_remove_policy(PG_FUNCTION_ARGS)
 {
-	Oid hypertable_oid = PG_GETARG_OID(0);
+	Oid table_oid = PG_GETARG_OID(0);
 	bool if_exists = PG_GETARG_BOOL(1);
+	Cache *hcache;
+
+	Hypertable *hypertable = ts_hypertable_cache_get_cache_and_entry(table_oid, true, &hcache);
+	if (!hypertable)
+	{
+		char *schema_name = get_namespace_name(get_rel_namespace(table_oid));
```

Comment:
return value of get_rel_namespace should be checked

debug=# select remove_drop_chunks_policy('1');
ERROR:  42704: no hypertable or continuous aggregate by the name "(null)" exists
LOCATION:  drop_chunks_remove_policy, drop_chunks_api.c:227
Time: 0.841 ms

Reply:
Fixed.